api/docs/dist/spec.json

```diff
@@ -3028,7 +3016,7 @@
         }
       }
     },
-    "/dc/comments/{dccid}": {
+    "/dc/comments/{dcid}": {
```

Comment:
Wouldn't `dccid` stand for `data collection comment ID` in this case, and therefore we should keep it? (These abbreviations/acronyms are immensely confusing...)

Reply:
Yep, I got confused by the fact that in the Synchweb code it is called dcid (https://github.com/DiamondLightSource/SynchWeb/blob/master/api/src/Page/DC.php#L1673) but it is still a data collection comment id.